src/pcapi/core/offerers/repository.py

```diff
@@ -0,0 +1,18 @@
+from typing import List
+
+from pcapi.models import Offerer
+from pcapi.models import UserOfferer
+
+
+def get_all_validated(
+    user_id: int,
+    user_is_admin: bool,
+) -> List[Offerer]:
+    query = Offerer.query.distinct()
+
+    if not user_is_admin:
+        query = query.join(UserOfferer, UserOfferer.offererId == Offerer.id).filter(UserOfferer.userId == user_id)
+
+    query = query.filter(Offerer.validationToken == None)
```

Comment:
```suggestion
    query = query.filter(Offerer.validationToken.is_(None))
```

(La version avec `==` lèverait un avertissement pylint si on ne l'avait pas [désactivé ici](https://github.com/pass-culture/pass-culture-api/blob/023874e14969e2c61ea336bc4b235aca9fd27f79/.pylintrc#L43-L46). Mais j'aimerais bien réactiver cet avertissement "bientôt".)

Reply:
Bien vu merci

Reply:
Je l'ai fait pour le reste de l'app en passant